src/assets/scss/_components.scss

```diff
@@ -364,41 +424,115 @@ em.ais-Highlight-highlighted {
     line-height: 24px;
     padding: 12px 18px;
   }
+
   .post-content {
-    & > a {
+    &>a {
       word-break: break-all;
     }
   }
+
   // make subtitle data have the same font size
   #subtitle {
     .post-content {
       display: flex;
       flex-wrap: wrap;
       gap: 0.5rem;
-      & > * {
+
+      &>* {
         margin: 0.5rem 0 0 0;
         font-size: 1.125rem;
       }
+
       // delete title from subtitle
       h2 {
         display: none;
       }
     }
   }
+
   time.article__meta-time {
     font-size: 0.8125rem;
     color: var(--text-lighter);
   }
+
   .gatsby-image-wrapper {
     img {
       filter: drop-shadow(0px 2px 8px rgba(0, 0, 0, 0.08));
     }
   }
-  .maxHeight400 > div > div {
+
+  .maxHeight400>div>div {
     height: 400px;
     object-position: top;
   }
+
   // li.subcategoryfolder {
   //   margin-left: 2.8rem;
   // }
 }
+
+// Cookie notice
+.CookieConsent {

```

Comment:
Optional: Pascal case seems fairly unusual for CSS names. Is this a convention in the KC? (Interestingly, the class above uses camel case, the one above kebab case…)